### ISSUE TITLE
Sorting, getting item with max value

### ISSUE THREAD

**Issue by popraducj** [alexandre-spieser/mongodb-generic-repository#5]
Is there a way to query the db with your repository in order to get the item with max field value.
For exemple:
you have a colection of stuff: { username, stuffName, orderColumn} 
I would like to get the max order number for userName X. I've saw that the driver would support a findOne sortby : {orderColumn  :-1} but I cannot figure a way to do that with this repository. 

Can you please provide and example? Can you also document this for future users?

**Comment by alexandre-spieser** [comment]
Hello, you could try that (untested).
```
        public async Task<TDocument> GetByMaxAsync<TDocument, TKey>(Expression<Func<TDocument, bool>> filter, Expression<Func<TDocument, object>> orderByDescending)
            where TDocument : IDocument<TKey>
            where TKey : IEquatable<TKey>
        {
            return await GetCollection<TDocument, TKey>().Find(Builders<TDocument>.Filter.Where(filter))
                                                         .SortByDescending(orderByDescending)
                                                         .FirstOrDefaultAsync();
        }
```

**Comment by alexandre-spieser** [comment]
By the way, this has been added and released in version [1.3.7](https://github.com/alexandre-spieser/mongodb-generic-repository/releases/tag/1.3.7) 👍.
